wagtailio/services/templates/services/blocks/link_button.html

```diff
@@ -0,0 +1,9 @@
+<a href="{{ value.link }}" class="link-button">
+    {% if value.lead_text %}
+        <span class="link-button__lead">{{value.lead_text}} </span>
+    {% endif %}
+    <strong >{{value.action_text}}</strong>
+    <svg class="external-link-icon" viewBox="0 0 100 100">
+        <use href="#external-link">
```

Comment:
suggestion: `xlink:href` may be better to use here for browser support, but worth double checking

Reply:
I replaced this in all locations where I used `<use href="..." >` before. Did not notice any change on the page.